app/facades/foods_facade.rb

```diff
@@ -0,0 +1,11 @@
+class FoodsFacade
+  def self.food_with(ingredient)
```

Comment:
So here is where we can make one method that gets us all that we want/need. We can set line 3 to a variable 
```
food_results = FoodService.food_with(ingredient)
```
Then we will iterate over our new variable and set this to another variable
```
food_attrs = food_results[:foods].map do |food_data|
Food.new(food_data)
```
Notice I took out the `[0..9]` range because you already limited your results to 10 with `pageSize=10` in your Service.

Finally, the last line that we want returned, outside of the block is to create a nested array that has both totalHits and all the attributes that we want.
```
total_hits, foods = [food_results[:totalHits], food_attrs]
```
This will give us the ability to have access to both things in our Controller